src/attached_probe.h

```diff
@@ -41,11 +41,14 @@ class AttachedProbe
   void attach_software();
   void attach_hardware();
   void attach_watchpoint(int pid, const std::string &mode);
+  void attach_kfunc(void);
+  int detach_kfunc(void);
 
   Probe &probe_;
   std::tuple<uint8_t *, uintptr_t> func_;
   std::vector<int> perf_event_fds_;
   int progfd_ = -1;
+  int tracing_fd_ = -1;
```

Comment:
For suppressing warning about unused variable if kfunc is not available.
```suggestion
#ifdef HAVE_KFUNC
  int tracing_fd_ = -1;
#endif 
```

Reply:
right, will add